pkg/flare/archive.go

```diff
@@ -157,6 +157,9 @@ func createArchive(fb flarehelpers.FlareBuilder, confSearchPaths SearchPaths, lo
 
 	fb.RegisterFilePerm(security.GetAuthTokenFilepath())
 
+	fb.RegisterDirPerm(filepath.Dir(config.Datadog.GetString("system_probe_config.sysprobe_socket")))
```

Comment:
On Windows the socket is a address (the configuration as `localhost:3333` for example).

Reply:
added a check for linux